eclipse-collections-api/src/main/java/org/eclipse/collections/api/bimap/ImmutableBiMap.java

```diff
@@ -134,6 +139,22 @@
         return this.groupByUniqueKey(function, target).toImmutable();
     }
 
+    @Override
+    default <KK, VV> ImmutableMap<KK, VV> aggregateInPlaceBy(
```

Comment:
Hi @motlin, please add since tags as you did for aggregateBy.

Reply:
done